### ISSUE TITLE
 NavigationDuplicated error when clicking on template

### ISSUE THREAD

**Issue by priamai** [johnsusek/praeco#298]
Hi there,
I would like to know if we can safely ignore this error from the browser web console when clicking any template:

```
vue-router.esm.js:1958 Uncaught (in promise) NavigationDuplicated: Avoided redundant navigation to current location: "/templates/detection_template".
    at Ye (http://192.168.2.16:8080/js/chunk-vendors.1c5e8f8c.js:218:17071)
    at qe (http://192.168.2.16:8080/js/chunk-vendors.1c5e8f8c.js:218:16650)
    at t.et.confirmTransition (http://192.168.2.16:8080/js/chunk-vendors.1c5e8f8c.js:218:20858)
    at t.et.transitionTo (http://192.168.2.16:8080/js/chunk-vendors.1c5e8f8c.js:218:20145)
    at t.push (http://192.168.2.16:8080/js/chunk-vendors.1c5e8f8c.js:218:22590)
    at http://192.168.2.16:8080/js/chunk-vendors.1c5e8f8c.js:218:27562
    at new Promise (<anonymous>)
    at xt.push (http://192.168.2.16:8080/js/chunk-vendors.1c5e8f8c.js:218:27525)
    at o.handleSelect (http://192.168.2.16:8080/js/app.d23a7d6f.js:1:339309)
    at it (http://192.168.2.16:8080/js/chunk-vendors.1c5e8f8c.js:68:11664)
```

See screenshot below
![image](https://user-images.githubusercontent.com/57333254/103475835-a7cce680-4da8-11eb-9629-b461ddeb2907.png)

Praeco version: 1.5.1
ES version: 7.6.2

Happy new year!

**Comment by nsano-rururu** [comment]
I have never used this function. I will investigate from now on

**Comment by nsano-rururu** [comment]
https://github.com/johnsusek/praeco/pull/253
Since the only modification of 253 was rules, I wonder if the same modification is necessary for template.

src/components/NavTree.vue 
src/views/TemplateView.vue

**Comment by nsano-rururu** [comment]
The same fix as 253 will fix the error, but I still get the error when renaming. The same is true when renaming rules.

**Comment by nsano-rururu** [comment]
It looks like it will still take some time to fix.

**Comment by nsano-rururu** [comment]
Released a version "praecoapp/praeco:1.5.3" and "praecoapp/elastalert-server:20210104"
https://hub.docker.com/r/praecoapp/praeco
https://hub.docker.com/r/praecoapp/elastalert-server

**Comment by priamai** [comment]
Wow that was fast I am going to test your fix.
